src/lib/sparse_ledger_lib/sparse_ledger.ml

```diff
@@ -192,16 +262,24 @@ let%test_module "sparse-ledger-test" =
         digest_string
           (sprintf "sparse-ledger_%03d" height ^ to_binary x ^ to_binary y)
 
-      let gen = Quickcheck.Generator.map String.gen ~f:digest_string
+      let gen = Quickcheck.Generator.map String.gen ~f:Md5.digest_string
     end
 
     module Account = struct
-      module T = struct
-        type t = {name: string; favorite_number: int}
-        [@@deriving bin_io, eq, sexp]
+      module Stable = struct
+        module V1 = struct
+          module T = struct
+            type t = {name: string; favorite_number: int}
+            [@@deriving bin_io, eq, sexp, version]
+          end
+
+          include T
+        end
+
+        module Latest = V1
       end
 
-      include T
+      include Stable.V1
```

Comment:
In general, we don't want to `include` versioned types, see comment below for `Key`.